umbral/utils.py

```diff
@@ -42,3 +44,23 @@ def kdf(ecpoint, key_length):
 
 def get_curve_keysize_bytes(curve):
     return int(math.ceil(curve.key_size / 8.00))
+
+
+class AbstractFromToBytes(ABC):
```

Comment:
Hmm - This name needs is too literal (We dont call a car a wheel!).  I'm not sure what an "AbstractFromToBytes" is.  In past team conversations about high level objects, we discussed "CryptoEntity", thoughts?

Reply:
I have called it `AbstractCryptoEntity` for now. I am not familiar with the domain and will go with whatever name seems best to the team